package.json

```diff
@@ -30,7 +30,7 @@
     "prepublish": "mastarm prepublish lib:build",
     "cover": "npm test -- --coverage",
     "lint": "mastarm lint lib",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "semantic-release": "semantic-release",
```

Comment:
It looks like this package script wasn't being used in the Travis CI run. Is that your understanding? I don't fully know what these removed steps used to do, but perhaps @evansiroky can weigh in on whether they're important.

Reply:
Correct, Travis was invoking `semantic-release` directly, which fails with GH actions (unless I missed something).